File: salimmo-parent/property-service/src/main/java/org/propertyservice/dto/PropertyLocationDto.java
package org.propertyservice.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.*;


@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
public class PropertyLocationDto {
    private Long id;
    private PropertyDto property;
    @JsonIgnoreProperties("propertyLocation")
    private CityDto city;
    private int houseNumber;
    private String street;
    private String latitude;
    private String longitude;
}

File: salimmo-frontend/src/app/models/propertylocation/propertyLocationDto.ts
import {PropertyDto} from "../property/propertyDto";
import {CityDto} from "../city/cityDto";

export class PropertyLocationDto {
  id!:number;
  property!:PropertyDto;
  city!:CityDto;
  houseNumber!:number;
  street!:string;
  latitude!:string;
  longitude!:string;
}
